mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -551,7 +721,7 @@
             <LayoutButtons index="3" Face="NukeArm" AbilCmd="ArmSiloWithNuke,Ammo1" Column="0"/>
             <LayoutButtons index="4" Face="SelectBuilder" Type="SelectBuilder" AbilCmd="" Row="1" Column="3"/>
             <LayoutButtons index="5" Face="ResearchPersonalCloaking" AbilCmd="GhostAcademyResearch,Research1" Column="0"/>
-            <LayoutButtons index="6" Face="ResearchEnhancedShockwaves" Type="AbilCmd" AbilCmd="GhostAcademyResearch,Research3" Row="0" Column="1"/>
+            <LayoutButtons index="6" removed="1"/>
```

Comment:
Enhanced Shockwaves upgrade removed.

The button to research the upgrade has been removed.
Hackers and AI could still research it, though. Thus, I dislike this solution